src/main.py

```diff
@@ -1,29 +1,93 @@
-# __author__ = "Vasudev Gupta"
-# __author_email__ = "7vasudevgupta@gmail.com"
-
 """Run this script to launch training"""
 
+from dataclasses import dataclass, replace
+
 import tensorflow as tf
 
-from trainer import Trainer
-from wav2vec2 import Wav2Vec2ForCTC
-from functools import partial
+from data_utils import LibriSpeechDataLoader, LibriSpeechDataLoaderArgs
+from wav2vec2 import CTCLoss, Wav2Vec2ForCTC
 
-from .data_utils import DataLoader
 
-MODEL_ID = "wav2vec2-base"
+@dataclass
+class TrainingArgs:
+    lr: float = 7e-5
+    max_epochs: int = 2
+    batch_size: int = 8
+    apply_spec_augment: bool = False
 
+    audio_maxlen: int = 50000
+    labels_maxlen: int = 128
 
-if __name__ == "__main__":
+    seed: int = 0
+
+    train_dir: str = "../data/LibriSpeech/test-clean"
+    val_dir: str = "../data/LibriSpeech/test-clean"
+    test_dir: str = "../data/LibriSpeech/test-clean"
```

Comment:
Ditto.

Reply:
Sorry, I am unable to follow here. Since these paths are already in args, do we need to shift these paths to separate script?

Reply:
I see. If these paths are a part of `args` then we should try accessing them via `args` only unless I am missing something?

Reply:
I am accessing them via args only (see [this](https://github.com/vasudevgupta7/gsoc-wav2vec2/blob/c01b659c03b2d3ac13ad0b57baeb6f64d40b0a39/src/main.py#L38))